orcas/l1l2.go

```diff
@@ -489,6 +572,10 @@ func (l *L1L2Orca) Get(req common.GetRequest) error {
 		}
 	}
 
+	// finish up metrics for overall L1 (batch) get operation
```

Comment:
Is this a batch get?  Does evcache-client always do a batch get or is it rare?

This code measures the entire batch.  Could we get a per query number instead?